src/project.ts

```diff
@@ -8,12 +8,19 @@ export type Urn = string;
 export type Peer = string;
 export type Branch = { [key: string]: string };
 
-export interface ProjectListing {
-  name: string;
-  urn: Urn;
+export interface Anchor {
+  confirmed: true;
```

Comment:
I understand the idea behind this new property, but I'm not sure where we are using it. Since we already have the `confirmations` array on the `PendingAnchor` type.